openidm-maintenance/src/test/java/org/forgerock/openidm/maintenance/upgrade/BundleHandlerTest.java

```diff
@@ -142,131 +142,132 @@ private void startBundle(Bundle bundle) throws Exception {
-
-        // at the "system" level remove the bundle from the bundles/directory now
-    }
-
-    @Test
-    public void testInstallBundle() throws Exception {
-        // create bundle info for installing
-        Bundle bundle = installedBundles.get(bundlePath);
-
-        // check that the bundle does not exist already in the framework
-        assertThat(bundle).isNull();
-
-        // install the bundle
-        bundle = installBundle(bundlePath);
-
-        // validate that the bundle has actually been installed
-        assertThat(bundle).isNotNull();
-        assertThat(bundle.getState()).isEqualTo(Bundle.INSTALLED);
-
-    }
-
-    @Test
-    public void testStartBundle() throws Exception {
-        // install bundle
-        Bundle bundle = installBundle(bundlePath);
-
-        // start the bundle
-        startBundle(bundle);
-
-        // assert that the bundle is now started in the Active state
-        assertThat(bundle.getState()).isEqualTo(Bundle.ACTIVE);
-
-    }
-
-    /**
-     * This test updates a bundle that has the same version as
-     * the previous but contains a different class implementation.
-     */
-    @Test
-    public void testUpdateBundle() throws Exception {
-        Path backupFile = null;
-        try {
-            // Start API
-            testStartBundle();
-
-            // install bundle
-            Path bundlePathV1 =
-                    Paths.get(service.getProjectURI().resolve("bundle/HelloImplementation-1.0-SNAPSHOT.jar"));
-            Bundle bundle = installBundle(bundlePathV1);
-
-            // start the bundle
-            startBundle(bundle);
-
-            // assert that the bundle is now started in the Active state
-            assertThat(bundle.getState()).isEqualTo(Bundle.ACTIVE);
-
-            String bundle1Version = FileUtil.readManifest(bundlePathV1.toFile()).getValue("Bundle-Version");
-            assertThat(bundle1Version).isEqualTo(bundle.getVersion().toString());
-
-            Path bundlePathV2  =
-                    Paths.get(service.getProjectURI().resolve("bundle/updated/HelloImplementation-1.0-SNAPSHOT.jar"));
-
-            bundleHandler.upgradeBundle(bundlePathV2, "HelloImplementation");
-
-            // assert that the old file has been backed up with archive extension
-            backupFile = Paths.get(new URI(bundlePathV1.toUri().toString().concat(ARCHIVE_EXTENSION)));
-            assertThat(backupFile).isNotNull();
-
-            // update bundle
-            bundleHandler.updateBundle(bundle);
-
-            // refresh all bundles that would need the new import from bundle2
-            FrameworkWiring frameworkWiring = service.getSystemBundle().adapt(FrameworkWiring.class);
-            frameworkWiring.refreshBundles(null);
-        } finally {
-            // cleanup this test
-            if (backupFile != null) {
-                Files.delete(backupFile);
-            }
-        }
-    }
+//    @Test
```

Comment:
@wahyufrombandung i think it is ok to comment off the failed tests first. i saw that there are comments inside. can put a top comment to state why we are commenting off when commenting off the test?

Reply:
ok